Error: code=1500 [Replica(s) failed to execute write] message="Operation failed - received 1 responses and 1 failures" info={'failures': 1, 'received_responses': 1, 'required_responses': 2, 'consistency': 'ALL'}

Stacktrace:
{noformat}
  File "C:\tools\python2\lib\unittest\case.py", line 329, in run
    testMethod()
  File "D:\jenkins\workspace\cassandra-2.2_dtest_win32\cassandra-dtest\paging_test.py", line 1473, in test_ttl_deletions
    data = self.setup_data()
  File "D:\jenkins\workspace\cassandra-2.2_dtest_win32\cassandra-dtest\paging_test.py", line 1238, in setup_data
    'col3': int
  File "D:\jenkins\workspace\cassandra-2.2_dtest_win32\cassandra-dtest\datahelp.py", line 134, in create_rows
    query_results = execute_concurrent_with_args(session, prepared, [d.values() for d in dicts])
  File "C:\tools\python2\lib\site-packages\cassandra\concurrent.py", line 117, in execute_concurrent_with_args
    return execute_concurrent(session, list(zip(cycle((statement,)), parameters)), *args, **kwargs)
  File "C:\tools\python2\lib\site-packages\cassandra\concurrent.py", line 100, in execute_concurrent
    raise exc
'code=1500 [Replica(s) failed to execute write] message="Operation failed - received 1 responses and 1 failures" info={\'failures\': 1, \'received_responses\': 1, \'required_responses\': 2, \'consistency\': \'ALL\'}\n-------------------- >> begin captured logging << --------------------\ndtest: DEBUG: cluster ccm directory: d:\\temp\\dtest-pa47yw\n--------------------- >> end captured logging << ---------------------'
{noformat}

[History|http://cassci.datastax.com/view/cassandra-2.2/job/cassandra-2.2_dtest_win32/64/testReport/junit/paging_test/TestPagingWithDeletions/test_ttl_deletions/history/]. Looks to be a 1/10 failure rate or so, so it's infrequent.